runtime-sdk/modules/evm/src/precompile/mod.rs

```diff
@@ -128,6 +128,7 @@ impl<Cfg: Config, B: EVMBackendExt> PrecompileSet for Precompiles<'_, Cfg, B> {
             // Oasis-specific, general.
             (1, 1, 1) => sha512::call_sha512_256(handle),
             (1, 1, 2) => subcall::call_subcall(handle, self.backend),
+            (1, 1, 3) => sha512::call_sha512(handle),
```

Comment:
You could also move it one place up, as the subcalls haven't been deployed yet :)

Reply:
I'd need to recompile the subcall solidity example :sob:

Reply:
No need to. Thought it was simple.

Reply:
There is now a makefile so as long as you have `solc` installed it should be simple.

Reply:
Ok, updated.